src/include/framework/cmd/types.h

```diff
@@ -37,6 +37,7 @@ struct cmd_mod {
 };
 
 static inline const struct mod *cmd2mod(const struct cmd *cmd) {
+	if (!cmd) return NULL;
```

Comment:
I think You should check this before calling cmd2mod

Reply:
I did this so that I don't have to perform this check in the exec_call() function.

Reply:
Reverted